### ISSUE TITLE
Allow Installation of Multiple Mods in a Queue

### ISSUE THREAD

**Issue by WubTheGame** [ModOrganizer2/modorganizer#818]
**Is your feature request related to a problem? Please describe.**
Haven't been making issues for projects on git for a while, something popped up and I said nah, Github decided to wipe everything I had typed. So I'm gonna keep this short and sweet.

I do not like that I have to manually and individually install every mod in the downloads folder/tab. I strongly prefer the way NMM does it, and, as a matter of fact, I just switched over from NMM!

To have downloaded 300+ mods, and then have to click several times that many in order to get them all installed... You bulk put mods into the downloads tab, so why not bulk installation too?

All I can say is, I can't even.

**Describe the solution you'd like**
Allow users to right click on a mod in the downloads tab and have a menu option to install all mods not currently installed.

**Describe alternatives you've considered**
Allow either, both, or one of these options:
CTRL Clicking mods to select them for installation
CTRL+A for installation.
So, basically, allow multiple mods to be selected at the same time in the downloads tab (Again, like NMM).

**Additional context**
I *would* extract all my mod archives into the mods folder, but, the thing is, I have several FOMOD installers, and I don't wanna have to go through my mod list re-checking which ones have those and which don't, and then extracting them into the mods folder or whatever.

Please, please, make this a feature, It'd be a lifesaver for everyone.

(P.S to those that agree with me, you don't have to post saying so. Use Github's reaction system to give me a thumbs up instead. It saves me hope that those notifications are those of the developers answering my wishes.)
